### ISSUE TITLE
Override configs bearing the same name

### ISSUE THREAD

**Issue by varemenos** [creynders/load-grunt-configs#23]
Let me explain the reasoning behind this issue whose theme is basically modularity and code-reuse.

The goal is to have a npm package that contains the base of all the grunt task configurations so I don't have to recreate them in each new project, it also makes it very easy to re-use them so I don't have to keep going back to old projects and manually copy/paste the new changes. This helps structure big code bases and maintain large projects that are utilize the same grunt configurations.

And now let me walk you through what I'm trying to do:

first I'm trying to load task configurations from 2 directories like this:

```js
    var configs = require('load-grunt-configs')(grunt, {
        config: {
            src: [
                'node_modules/my-default-tasks-package/tasks/config/*.js',
                'tasks/config/*.js'
            ]
        }
    });
```

In  `node_modules/my-default-tasks-package/tasks/config/` I have all the "default" tasks used across multiple projects.
In `tasks/config/` I have the tasks for this particular project.

What I expected/wished to happen was for load-grunt-configs to load all the  tasks from `./node_modules` and afterwards the ones from `./tasks`. If any of the configurations of `./tasks` already existed in `./node_modules` then they would replace the ones in `./node_modules` (or at least merge them gracefully without combining everything together).

Instead what I noticed is, load-grunt-configs used `grunt.file.expand` to fetch all the files, then for each of them start placing them in the options variable by using `_.merge`. So in the end the tasks from inside `./node_modules` were merged with the ones in `./tasks`.

Basically, it all boils down to prevent load-grunt-configs from merging values below the subtask level.

Please let me know if this makes sense to you and whether it's possible to achieve, also sorry for making you read this big wall of text.

**Comment by creynders** [comment]
Hi there, your use case makes sense. I'm not 100% sure, but I think you can solve this already using the `mergeCustomizer`: https://github.com/creynders/load-grunt-configs#options and https://lodash.com/docs#merge
Basically you can decide what value needs to be returned: `a`, `b`, or `a` merged with `b`.

**Comment by varemenos** [comment]
Yep, returning `b` is pretty much what i wanted, thank you very much!
`mergeCustomizer` seems pretty great, having the ability to handle data on will is pretty powerful.

I will go ahead and close this, thank you for your time
